plugins/woocommerce/tests/e2e-pw/tests/shopper/product-tags-attributes.spec.js

```diff
@@ -317,7 +317,13 @@ test.describe(
 				.click();
 
 			await expect(
-				page.getByRole( 'button', { name: 'Update', exact: true } )
+				// WP 6.6 updates the button text from "Update" to "Save", so we'll need to check for either.
+				page.getByRole( 'button', { name: 'Update', exact: true } ).or(
```

Comment:
That should work. I preferred using Playwright builtin `locator or` for better error handling, but I think in this case it will work just the same. I'll try it our in a follow-up. Good improvement.